Add docs for 'delete unenrolled agents' (#1383)
elastic/ingest-docs: docs/en/ingest-management/fleet/fleet-settings.asciidoc

You can specify a proxy server to be used in {fleet-server}, {agent} outputs, or for any agent binary download sources. 
For full details about proxy configuration refer to <<fleet-agent-proxy-support,Using a proxy server with {agent} and {fleet}>>.

[discrete]
[[delete-unenrolled-agents-setting]]
== Delete unenrolled agents

After an {agent} has been unenrolled in {fleet}, a number of documents about the agent are retained just in case the agent needs to be recovered at some point. You can choose to have all data related to an unenrolled agent deleted automatically.

> Note that this option is enabled only if the `xpack.fleet.enableDeleteUnenrolledAgents` setting is not already set to `false` in the {kibana-ref}/[{kib} settings file].

Reviewer: ```suggestion
Note that this option can also be enabled by adding the `xpack.fleet.enableDeleteUnenrolledAgents: true` setting to the {kibana-ref}/[{kib} settings file].
```

Author: Thanks Julia!
